index.js

```diff
@@ -43,15 +29,15 @@ module.exports = () => {
       }
 
       if (type === 'IPFS_INIT_FINISHED') {
-        return Object.assign({}, state, { ready: true, identity: payload })
+        return Object.assign({}, state, { ready: true, ...payload })
```

Comment:
I don't think we can use spread without adding a transpilation step to this repo

Reply:
@olizilla it should work after 8.6.0 (https://node.green/#ES2018-features-object-rest-spread-properties).

Are we using any older version?

Reply:
I'm more thinking that we are publishing this as an npm module, and it's worth thinking about how people might want to consume it.

It's intended to be used in browsers rather than node, so it needs some thought.

Reply:
@olizilla okay. I'll change that then 😄 I just saw the compatibility table on MDN and I really thought Edge and Safari worked with object spread but they don't!